tests/Loader/LoaderIntegrationTest.php

```diff
@@ -581,6 +583,43 @@ public function testLoadSelfReferencedFixtures()
         $this->assertCount(2, $objects);
     }
 
+    public function testLoadRangeReference()
+    {
+        $data = [
+            User::class => [
+                'user0' => [
```

Comment:
what happens if we have a template user, e.g. `base_user (template)` and the `user0` would extend it? (I wouldn't expect the `base_user` to be picked up)

Reply:
Good point, skipped now template fixtures when building the list.